Homework13/Tests/DemoQA/Elements/DynamicPropertiesTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Homework13.Common.Drivers;
+using Homework13.Data;
+using Homework13.Data.Constants;
+using Homework13.PageObjects.DemoQA;
+using NUnit.Framework;
+
+namespace Homework13.Tests.DemoQA.Elements
+{
+    public class DynamicPropertiesTests : BaseTest
+    {
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            WebDriverFactory.Driver.Navigate().GoToUrl(TestSettings.DemoQaElementsUrl);
+            GenericPages.BaseDemoQaPage.ClickOnElementsSubCategory(ElementsCategories.DynamicProperties);
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            WebDriverFactory.Driver.Navigate().Refresh();
+        }
+
+        [Test]
+        public void WaitForButtonTest()
+        {
+            // Waiter with 15 seconds timeout by default is already applied in WebDriverExtensions
+            // Check that button is displayed
+            Assert.IsTrue(GenericPages.DynamicPropertiesPage.IsButtonToWaitForDisplayed());
+        }
+
+        [Test]
+        public void DangerRedTextButtonTest()
+        {
+            const string RedColor = "rgba(220, 53, 69, 1)";
```

Comment:
Naming - variables (names) of methods always starts with lowerCase letter

Reply:
fixed